extensions/objects/models/objects_test.py

```diff
@@ -762,6 +762,16 @@ def test_algebraic_identifier_validation(self):
             objects.AlgebraicIdentifier, mappings,
             invalid_values_with_error_messages)
 
+    def test_ratio__validation(self):
+        """Tests objects of type RatioExpression."""
+
+        mappings = [('a', 'a'), ('alpha', 'alpha'), ('Z', 'Z')]
```

Comment:
You need to test with rations, there aren't ratios.

Reply:
done